node/plain/webappWithTests/Application/Tests/functional_tests/sampleFunctionalTests.js

```diff
@@ -0,0 +1,38 @@
+const process = require('process');
+const webdriver = require('selenium-webdriver');
+const {until} = require('selenium-webdriver');
+
+process.env['Path'] = process.env['Path'] + ';' + process.env['ChromeWebDriver'];

```

Comment:
how are we getting ChromeDriver to that folder?

Reply:
it is already there on the hosted machines